rust/src/delta.rs

```diff
@@ -869,7 +892,7 @@ impl DeltaTable {
 
     /// Returns a vector of tombstones (i.e. `Remove` actions present in the current delta log.
     pub fn get_tombstones(&self) -> &Vec<action::Remove> {
-        &self.state.tombstones
+        self.state.all_tombstones()
```

Comment:
could we mark this method as deprecated in favor of the new `all_tombstones` method?

Reply:
The `get_tombstones`? I wonder which one we should expose to the table level? I lean toward 
table level -> non stale tombstones
state level -> both all and non stale

Reply:
oh, my bad, i thought `all_tombstones` is a method of DetltaTable, not the state. I agree with you that we can keep this method and change it to return active tombstones.

Reply:
done